content/en/user-guide/aws/apigateway/index.md

```diff
@@ -0,0 +1,389 @@
+
+{{< command >}}
+$ awslocal apigateway get-resources --rest-api-id <REST_API_ID>
+{{< /command >}}
+
+The above command returns the following response:
+
+```json
+{
+    "items": [
+        {
+            "id": "u53af9hm83",
+            "path": "/"
+        }
+    ]
+}
+```
+
+Note the ID of the root resource returned in the response. You'll need this ID for the next step.
+
+### Create a Resource
+
+Create a new resource for the API using the [`CreateResource`](https://docs.aws.amazon.com/apigateway/latest/api/API_CreateResource.html) API. Use the ID of the resource returned in the previous step as the parent ID:
+
+{{< command >}}
+$ awslocal apigateway create-resource \
+  --rest-api-id <REST_API_ID> \
+  --parent-id <PARENT_ID> \
+  --path-part "{somethingId}"
+{{< /command >}}
+
+The above command returns the following response:
+
+```json
+{
+    "id": "zzcvcf56ar",
+    "parentId": "u53af9hm83",
+    "pathPart": "{somethingId}",
+    "path": "/{somethingId}"
+}
+```
+
+Note the ID of the root resource returned in the response. You'll need this Resource ID for the next step.
+
+### Add a method and integration
+
+Add a `GET` method to the resource using the [`PutMethod`](https://docs.aws.amazon.com/apigateway/latest/api/API_PutMethod.html) API. Use the ID of the resource returned in the previous step as the Resource ID:
+
+{{< command >}}
+awslocal apigateway put-method \
+  --rest-api-id <REST_API_ID> \
+  --resource-id <RESOURCE_ID> \
+  --http-method GET \
+  --request-parameters "method.request.path.somethingId=true" \
+  --authorization-type "NONE"
+{{< /command >}}
+
+The above command returns the following response:
+
+```json
+{
+    "httpMethod": "GET",
+    "authorizationType": "NONE",
+    "apiKeyRequired": false,
+    "requestParameters": {
+        "method.request.path.somethingId": true
+    }
+}
+```
+
+Now, create a new integration for the method using the [`PutIntegration`](https://docs.aws.amazon.com/apigateway/latest/api/API_PutIntegration.html) API.
+
+{{< command >}}
+$ awslocal apigateway put-integration \
+  --rest-api-id <REST_API_ID> \
+  --resource-id <RSOURCE_ID> \
+  --http-method GET \
+  --type AWS_PROXY \
+  --integration-http-method POST \
+  --uri arn:aws:apigateway:us-east-1:lambda:path/2015-03-31/functions/arn:aws:lambda:us-east-1:000000000000:function:apigw-lambda/invocations \
+  --passthrough-behavior WHEN_NO_MATCH
+{{< /command >}}
+
+The above command integrates the `GET` method with the Lambda function created in the first step. We can now proceed with the deployment before invoking the API.
+
+### Create a deployment
+
+Create a new deployment for the API using the [`CreateDeployment`](https://docs.aws.amazon.com/apigateway/latest/api/API_CreateDeployment.html) API:
+
+{{< command >}}
+awslocal apigateway create-deployment \
+  --rest-api-id <REST_API_ID> \
```

Comment:
Should we try to make the commands fully copy/pasteable top to bottom? I.e., we could assign a variable `$restApiId` from the output above, and then use it here as a reference..

Reply:
Will take this up on a second iteration 👍